src/drawing/DrawingList.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import classNames from 'classnames';
+import cloneDeep from 'lodash/cloneDeep';
+import noop from 'lodash/noop';
+import DrawingTarget from './DrawingTarget';
+import useOutsideEvent from '../common/useOutsideEvent';
+import { AnnotationDrawing, PathGroup, Shape } from '../@types';
+import { checkValue } from '../utils/util';
+import { getShape } from './drawingUtil';
+
+export type Props = {
+    activeId?: string | null;
+    annotations: AnnotationDrawing[];
+    className?: string;
+    onSelect?: (annotationId: string | null) => void;
+};
+
+export const DRAWING_TARGET_PADDING = 10;
+
+export function filterDrawing({ target: { path_groups: pathGroups } }: AnnotationDrawing): boolean {
+    return pathGroups.reduce<boolean>(
+        (prevGroups, { paths }) =>
+            prevGroups &&
+            paths.reduce<boolean>(
+                (prevPaths, { points }) =>
+                    prevPaths &&
+                    points.reduce<boolean>(
+                        (prevPoints, { x, y }) => prevPoints && checkValue(x) && checkValue(y),
+                        true,
+                    ),
+                true,
+            ),
+        true,
+    );
+}
+
+export function scaleDrawing(
```

Comment:
Can we accomplish this via CSS `scale` transforms to avoid having to do all of this math ourselves? It would also remove the need for the "root dimensions" state we have below.

Reply:
That's an interesting idea, though I wonder if the stroke width would get affected by such a scale transform as well

Reply:
@ConradJChan you're right
![image](https://user-images.githubusercontent.com/17711683/98427559-8bcff180-2052-11eb-81db-35a139dc7865.png)